dask-gateway-server/dask_gateway_server/backends/kubernetes/controller.py

```diff
@@ -285,6 +344,18 @@ def _validate_proxy_prefix(self, proposal):
         config=True,
     )
 
+    make_ingressroute = Callable(
+        _make_ingressroute,
+        help="A callable function to override the default traefik ingressroute resource",
+        config=True,
+    )
+
+    make_ingressroutetcp = Callable(
+        _make_ingressroutetcp,
+        help="A callable function to override the default traefik ingressroutetcp resource",
+        config=True,
+    )
+
```

Comment:
I'd like a brief iteration on this helm string. This configuration is to override the default instance function on the KubeController class that takes a Dask cluster name and a namespace and returns a string representing a k8s manifest.

I guess what I'm missing most from the current description is the fact that it should be a callable function taking `self` (KubeController), clustername, namespace, and returns a string representing a valid k8s resource manifest.